src/styles/main.scss

```diff
@@ -0,0 +1,11 @@
+@import "utils/variables";
+@import "utils/mixins";
+@import "blocks/clock";
+
+* {
+  box-sizing: border-box;
```

Comment:
When I remove this `box-sizing` property nothing changes, so it's probably redundant here. But I wouldn't call it a mistake.